File: app/src/main/java/deltazero/amarok/ui/SecurityAuthForQSActivity.java
package deltazero.amarok.ui;

import deltazero.amarok.core.Hider;

public class SecurityAuthForQSActivity extends SecurityAuthActivity {
  @Override
  protected void onSuccess() {
    Hider.processAll(this, Hider.Action.UNHIDE);
    super.onSuccess();
  }
}

File: app/src/main/java/deltazero/amarok/ui/SecurityAuthForQSActivity.kt
package deltazero.amarok.ui

import deltazero.amarok.core.Hider

class SecurityAuthForQSActivity : SecurityAuthActivity() {
  override fun onSuccess() {
    Hider.processAll(this, Hider.Action.UNHIDE)
    super.onSuccess()
  }
}
